packages/next/client/image.tsx

```diff
@@ -410,6 +411,30 @@ export default function Image({
         ref={setRef}
         style={imgStyle}
       />
+      {priority ? (
+        // Note how we omit the `href` attribute, as it would only be relevant
+        // for browsers that do not support `imagesrcset`, and in those cases
+        // it would likely cause the incorrect image to be preloaded.
+        //
+        // https://html.spec.whatwg.org/multipage/semantics.html#attr-link-imagesrcset
+        <Head>
+          <link
+            key={
+              '__nimg-' +
```

Comment:
```suggestion
              '__next_image_' +
```

Do we have a convention here?

Reply:
No convention, was just trying to something short to avoid a collision. The user shouldn't really ever see this. Our normal convention is `_n_------------` though.